voc/python/ast.py

```diff
@@ -463,7 +463,12 @@ def visit_For(self, node):
         )
 
         loop = START_LOOP()
-
+        self.context.add_opcodes(
+            java.New('org/python/types/Bool'),
+            JavaOpcodes.ICONST_1(),
+            java.Init('org/python/types/Bool', 'Z'),
+            ASTORE_name('#for-orelse-%x' % id(loop))
```

Comment:
Hi! Last update from @fedeisas gets rid of the full object and stores an integer instead.